lisa/messages.py

```diff
@@ -263,6 +263,15 @@ class ProvisionBootTimeMessage(MessageBase):
     information: Dict[str, str] = field(default_factory=dict)
 
 
+@dataclass
+class KernelBuildMessage(MessageBase):
+    type: str = "KernelBuild"
+    old_kernel_version: str = ""
+    new_kernel_version: str = ""
+    is_success: bool = False
+    error_message: str = ""
```

Comment:
should we keep the first bad commit?

Reply:
This message is used by Kernel Installer, The bad commit is detected by the Combinator.